.circleci/config.yml

```diff
@@ -10,31 +10,32 @@ jobs:
     <<: *defaults
     steps:
       - checkout
-
+      - restore_cache:
+          keys:
+            - v2-npm-{{ checksum "package-lock.json" }}
+            - v2-npm-
```

Comment:
Is this line required? If yes, where is cache saving for this key defined?

Reply:
Yes, it is recommended by CircleCI to keep that if checksum fails.
Ref.: https://circleci.com/docs/2.0/caching/